arbitrum-docs/launch-orbit-chain/concepts/arbos-upgrade.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'ArbOS upgrade on orbit chains'
```

Comment:
What do you think of re-forming this into a how to or potentially splitting it?

Reply:
👍

Reply:
> What do you think of re-forming this into a how to or potentially splitting it?

Reply:
👍

Reply:
@GreatSoshiant our [pattern guide is here](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#2309f7f5db284126929065ac3c56fef8) for reference; procedural instructions are a perfect candidate for the how-to content type. Some additional rationale / related TODOs are in [this PR](https://github.com/OffchainLabs/arbitrum-docs/pull/799#discussion_r1431788299). Need a hand refactoring these, or would you like to take a stab at it?

Reply:
That completely makes sense to me! @symbolpunk I was just drafting it to have the content in hand and reviewed by our Engs. I will follow procedural instructions pattern as you mentioned and will get back soon!
Thanks